app/config/environment.d.ts

```diff
@@ -19,6 +19,7 @@ export interface KeenConfig {
 
 declare const config: {
     WATER_BUTLER_ENABLED: boolean;
+    PREPRINT_ADAPTER_OPTIONS_ENABLED: boolean;
```

Comment:
I appreciate this idea, as it always takes me a few minutes to realize why my preprint detail page is broken 😄